### ISSUE TITLE
Develop Training for Food-101

### ISSUE THREAD

**Issue by steliosgrs** [ieee-sb-uniwa/milon-nutrition-training#13]
## Phase 1 
- [x] Research and gather the Food-101 dataset.
- [x] Preprocess the dataset for training (resizing, normalization)
- [x] Design and implement a convolutional neural network (CNN) architecture suitable for food image classification.
- [x] Train the CNN model on the Food-101 dataset using 3-5 classes

## Phase 2
- [x] Augmentation the dataset for training using TensorFlow
- [x] Train again with the same classes and compare the models.
